src/acme/pd/Ticket.java

```diff
@@ -69,56 +66,85 @@
     @Column(name = "NOTE")
     private String note;
 
+    public Ticket() {
+    }
+
+    public Ticket(Company company) {
+        this.setCreationDateTime(LocalDateTime.now());
+        this.setCompany(company);
+        this.generatePackageId();
+        this.setClerk(this.getCompany().getCurrentUser());
+
+        this.setCompany(company);
+        HashMap<UUID, Customer> customers = new HashMap<>(company.getCustomers());
+        // Set the pickup customer to a customer (Useful for UI)
+        Customer tmpCust = (Customer) customers.values().toArray()[0];
+        this.setPickupCustomer(tmpCust);
+        // Remove customer from map to be sure that pickup and destination are different
+        customers.remove(tmpCust.getId());
+        // Set destination customer to a customer (Useful for UI)
+        this.setDeliveryCustomer((Customer) customers.values().toArray()[0]);
+        this.setDeliveryTime(LocalDateTime.now().plusHours(6));
```

Comment:
`this.setDeliveryTime()` should be handled by the code @dee-dee-raye already wrote for us.

Reply:
She says that she didnt